Mathlib/LinearAlgebra/AffineSpace/AffineEquiv.lean

```diff
@@ -220,11 +220,11 @@ def Simps.apply (e : P₁ ≃ᵃ[k] P₂) : P₁ → P₂ :=
 #align affine_equiv.simps.apply AffineEquiv.Simps.apply
 
 /-- See Note [custom simps projection] -/
-def Simps.symmApply (e : P₁ ≃ᵃ[k] P₂) : P₂ → P₁ :=
+def Simps.symm_apply (e : P₁ ≃ᵃ[k] P₂) : P₂ → P₁ :=
```

Comment:
I don't understand exactly what these definitions do, but shouldn't this remain `symmApply`, because it is a definition and a lemma. Maybe there's a convention I don't know about

Reply:
The case-sensitive string `symmApply` currently only appears in this one file, while `symm_apply` appears in many, many files. I have chosen to go for consistency.